src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyBasedCallerArgumentCollection.java

```diff
@@ -125,4 +130,11 @@ public ReadThreadingAssembler createReadThreadingAssembler() {
     @Argument(fullName = MAX_MNP_DISTANCE_LONG_NAME, shortName = MAX_MNP_DISTANCE_SHORT_NAME,
             doc = "Two or more phased substitutions separated by this distance or less are merged into MNPs.", optional = true)
     public int maxMnpDistance = getDefaultMaxMnpDistance();
+
+    @Argument(fullName= FORCE_CALL_ALLELES_LONG_NAME, doc="The set of alleles for which to force genotyping regardless of evidence", optional=true)
```

Comment:
There are a bunch of places where the variables or test names or whatever say force call, but the arg doc here says force genotyping.  Very minor nit to pick, but it would be good to be consistent.

Reply:
done